src/NuGet.Services.Metadata.Catalog.Monitoring/Utility/LogEvents.cs

```diff
@@ -7,6 +7,8 @@ namespace NuGet.Services.Metadata.Catalog.Monitoring
 {
     public static class LogEvents
     {
+        public static EventId QueueMessageFatalFailure = new EventId(905, "Failed to process queue message");
```

Comment:
Order this line below by event ID?  Then it's easier to see what the next unused ID is.